File: AbstractFactory/Python/FurnitureStore/factory/victorian_furniture_factory.py
from products.chair.chair import Chair
from products.chair.victorian_chair import VictorianChair
from factory.furniture_factory import FurnitureFactory
from products.sofa import Sofa
from products.sofa import VictorianSofa
from products.table import Table
from products.table.victorian_table import VictorianTable


class VictorianFurnitureFactory(FurnitureFactory):
    def create_chair(self) -> Chair:
        return VictorianChair()

    def create_sofa(self) -> Sofa:
        return VictorianSofa()

    def create_table(self) -> Table:
        return VictorianTable()

File: AbstractFactory/Java/FurnitureStore/src/com/laxmena/factory/VictorianFurnitureFactory.java
package com.laxmena.factory;

import com.laxmena.chair.Chair;
import com.laxmena.chair.VictorianChair;
import com.laxmena.sofa.Sofa;
import com.laxmena.sofa.VictorianSofa;
import com.laxmena.table.Table;
import com.laxmena.table.VictorialTable;

public class VictorianFurnitureFactory implements FurnitureFactory {
    @Override
    public Chair createChair() {
        return new VictorianChair();
    }

    @Override
    public Sofa createSofa() {
        return new VictorianSofa();
    }

    @Override
    public Table createTable() {
        return new VictorialTable();
    }
}
